disk_objectstore/container.py

```diff
@@ -167,6 +167,10 @@ def __exit__(self, exc_type: Any, exc_value: Any, traceback: Any) -> None:
         """Close the session when exiting the context."""
         self.close()
 
+    def __del__(self) -> None:
+        """Closes all connections on deletion."""
+        self.close()
```

Comment:
Can you close the resources twice? For example, you already call close() and then you delete the container, will the `close` called again, what will happened?

Reply:
Should work, will add a test